src/http/response.rs

```diff
@@ -223,7 +223,7 @@ impl From<Response> for Vec<u8> {
 
     if !val.body.is_empty() {
       bytes.extend(val.body);
-      bytes.extend(b"\r\n");
+      //bytes.extend(b"\r\n");
```

Comment:
This could be removed. Not going to block on this though. When doing a fuzzing/coverage pass, I'm sure eyes will end up on this again.